src/components/resourceTable.js

```diff
@@ -166,15 +171,15 @@ class ResourceTable {
       }
       return 0;
     });
```

Comment:
why await, we want to remove the c right or is that just for testing?

Reply:
Left in temporarily for testing purposes